docs/sources/getting-started/_index.md

```diff
@@ -1,14 +1,139 @@
+    ```bash
+    mkdir evaluate-loki
+    cd evaluate-loki
+    ```
+1. Download `loki-config.yaml`, `promtail-local-config.yaml`, and `docker-compose.yaml`:
+
+    ```bash
+    wget https://raw.githubusercontent.com/grafana/loki/main/production/simple-scalable/loki-config.yaml -O loki-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/production/simple-scalable/promtail-local-config.yaml -O promtail-local-config.yaml
+    wget https://raw.githubusercontent.com/grafana/loki/main/production/simple-scalable/docker-compose.yaml -O docker-compose.yaml
+    ```
+1. Modify the Promtail configuration file to scrape the output of the flog app. Edit `promtail-local-config.yaml`. Change the `__path__` value `/path/to/flog.log` to be the path to your current working directory, which is the test environment's directory.
+
+    Within Promtail configuration, the `scrape_configs` YAML block specifies the logs to scrape.
+
+1. Download and install Promtail.
+    1. On the [Loki](https://github.com/grafana/loki) home page, locate and click on the most recent release.  At the bottom of the releases's page are the Assets.  Find and download the Promtail ZIP file for your operating system and architecture.
+    1. Uncompress the binary.  For example:
+        ```bash
+        unzip promtail-linux-amd64.zip
+        ```
+
+## Deploy the test environment
+
+All shell commands are issued from the test environment's directory.
+
+1. Deploy Grafana and the Loki cluster with `docker-compose`:
+    ```bash
+    docker-compose up
+    ```
+1. (Optional) Verify that the Loki cluster is up and running. The read component returns `ready` when you point a web browser at http://localhost:3101/ready.
+The write component returns `ready` when you point a web browser at http://localhost:3102/ready.
+1. Run the `flog` app to start generating log lines.
+This example generates 100 log lines in JSON format, at intervals of 5 seconds:
+    ```bash
+    flog -f json -o flog.log -t log -n 100 -d 5s
+    ```
+1. Run Promtail to capture the log lines and push them to the Loki cluster. Your binary's name may be different than this example:
+    ```bash
+    ./promtail-linux-amd64 -config.file promtail-local-config.yaml
+    ```
+
+## Use Grafana and the test environment
+
+This guide uses Grafana to query and observe the log lines captured in the Loki cluster.
+
+Use Grafana to query and observe the log lines captured in the Loki cluster by navigating a browser to http://localhost:3000.
+The Grafana instance has Loki configured as a [datasource](https://grafana.com/docs/grafana/latest/datasources/loki/).
+
+Click on the Grafana instance's [Explore](https://grafana.com/docs/grafana/latest/explore/) icon to bring up the log browser.
+
+Use Grafana to try some queries.
+Enter your query into the **Log browser** box, and click on the blue **Run query** button.
+
+To see all the log lines that flog has generated:
+```
+{job="flog"}
```

Comment:
This is where I get stuck. This query returns nothing:

![image](https://user-images.githubusercontent.com/100453168/164564040-c2fa7eb6-1773-4fa5-8215-032893ff534d.png)

`flog.log` is populating as expected:

```
$ tail flog.log
{"host":"70.118.136.217", "user-identifier":"-", "datetime":"21/Apr/2022:15:50:27 -0700", "method": "DELETE", "request": "/engage/incubate/enhance/deploy", "protocol":"HTTP/1.0", "status":400, "bytes":25672, "referer": "http://www.directturn-key.name/distributed/disintermediate/mesh/killer"}
{"host":"15.233.153.159", "user-identifier":"damore6414", "datetime":"21/Apr/2022:15:50:32 -0700", "method": "GET", "request": "/supply-chains", "protocol":"HTTP/1.0", "status":406, "bytes":9016, "referer": "https://www.futurewireless.name/impactful/productize"}
...
```

`promtail` sees `flog.log`:

```
level=info ts=2022-04-21T22:54:35.802272258Z caller=tailer.go:126 component=tailer msg="tail routine: started" path=/home/gguillotte/evaluate-loki/flog.log
```

Loki logs POST requests from `promtail`:

```
gateway_1   | 172.18.0.1 - - [21/Apr/2022:23:02:31 +0000] "POST /loki/api/v1/push HTTP/1.1" 204 0 "-" "promtail/"
write_1     | level=info ts=2022-04-21T23:02:31.991891325Z caller=table_manager.go:169 msg="uploading tables"
write_1     | level=info ts=2022-04-21T23:02:31.998237593Z caller=checkpoint.go:615 msg="starting checkpoint"
write_1     | level=info ts=2022-04-21T23:02:31.998617974Z caller=checkpoint.go:340 msg="attempting checkpoint for" dir=/loki/wal/checkpoint.000007
gateway_1   | 172.18.0.1 - - [21/Apr/2022:23:02:32 +0000] "POST /loki/api/v1/push HTTP/1.1" 204 0 "-" "promtail/2.5.0"
read_1      | level=info ts=2022-04-21T23:02:32.53353582Z caller=table_manager.go:209 msg="syncing tables"
gateway_1   | 172.18.0.1 - - [21/Apr/2022:23:02:32 +0000] "POST /loki/api/v1/push HTTP/1.1" 204 0 "-" "promtail/"
```

But I still don't get any results returned from the `{job="flog"}` query in Grafana. The query runs and reports `returned_lines=0` consistently:

```
read_1      | level=info ts=2022-04-21T23:10:12.272407327Z caller=metrics.go:122 component=frontend org_id=tenant1 latency=fast query="sum by (level) (count_over_time({job=\"flog\"}[1m]))" query_type=metric range_type=range length=1h0m0s step=1m0s duration=17.273355ms status=200 limit=1185 returned_lines=0 throughput=0B total_bytes=0B queue_time=238.214385ms subqueries=3 source=logvolhist
gateway_1   | 172.18.0.7 - - [21/Apr/2022:23:10:12 +0000] "GET /loki/api/v1/query_range?direction=BACKWARD&limit=1185&query=sum%20by%20(level)%20(count_over_time(%7Bjob%3D%22flog%22%7D%5B1m%5D))&start=1650557052033000000&end=1650560652033000000&step=60 HTTP/1.1" 200 772 "-" "Grafana/8.5.0"
```